R/process_extract_home_care.R

```diff
@@ -28,11 +28,13 @@ process_extract_home_care <- function(
 
   # Selections for financial year------------------------------------
 
-  hc_data <- data %>%
+  hc_data <- all_hc_processed %>%
```

Comment:
This will break here because the parameter is called `data` even though you are passing the `all_hc_processed` data through 
```suggestion
  hc_data <- data %>%
```